test_runner/fixtures/neon_fixtures.py

```diff
@@ -2436,10 +2438,15 @@ def remote_pg(
     connstr = os.getenv("BENCHMARK_CONNSTR")
     if connstr is None:
         raise ValueError("no connstr provided, use BENCHMARK_CONNSTR environment variable")
-
+    start_ms = int(datetime.timestamp(datetime.now(timezone.utc)) * 1000)
```

Comment:
```python
int(datetime.utcnow().timestamp()) * 1000
```

it's a bit shorter

Reply:
and maybe it's better to multiple by `1_000` before coercing to `int`?

Reply:
> ```python
> int(datetime.utcnow().timestamp()) * 1000
> ```
> 
> it's a bit shorter

Yep, I was going to, but then I was scared by a huge red warning https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow 😄

Reply:
Hmm, I never saw errors with this. Nonetheless, there is a method `timestamp` in `datetime` object